app/controllers/users/registrations_controller.rb

```diff
@@ -1,22 +1,22 @@
 # frozen_string_literal: true
-
 class Users::RegistrationsController < Devise::RegistrationsController
-
   include RackSessionsFix
   respond_to :json
 
-  private
-
-  def respond_with(current_user, _opts = {})
-    if resource.persisted?
-      render json: {
-        status: {code: 200, message: 'Signed up successfully.'},
-        data: UserSerializer.new(current_user).serializable_hash[:data][:attributes]
-      }
+  def create
+    contract = Contracts::Users::UserRegistrationContract.new
```

Comment:
You shouldn't be calling a contract class here.  You should be calling an operation class which has a `validate_params` method that establishes and verifies the contract before it attempts to process.